R/tar_load.R

```diff
@@ -50,3 +51,24 @@ tar_load <- function(
     store = store
   )
 }
+
+#' @export
+#' @rdname tar_load
+tar_load_all <- function(
+  branches = NULL,
+  meta = tar_meta(targets_only = TRUE, store = store),
+  strict = TRUE,
+  silent = FALSE,
+  envir = parent.frame(),
+  store = targets::tar_config_get("store")
+) {
+  tar_load(
```

Comment:
Would you include `force(envir)` here at the top? That way, `parent.frame()` refers to the parent frame of your function and not that of `tar_load()`.

Reply:
good point